Documentation/deploy-master.md

```diff
@@ -165,7 +165,7 @@ spec:
     - --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
     - --client-ca-file=/etc/kubernetes/ssl/ca.pem
     - --service-account-key-file=/etc/kubernetes/ssl/apiserver-key.pem
-    - --runtime-config=extensions/v1beta1=true,extensions/v1beta1/thirdpartyresources=true
+    - --runtime-config=extensions/v1beta1=true,extensions/v1beta1/networkpolicies=true
```

Comment:
Are thirdParty Resources enabled by default now? We still want these in general (even if not being used by networkpolicy)